app/models/db/Filters.scala

```diff
@@ -66,39 +67,38 @@ object MetricsFilters {
 
   private def getOptionFromQS(key: String, qs: Map[String, Seq[String]]): Option[String] = qs.get(key).flatMap(_.headOption)
 
-  def metricFilters(implicit filters: MetricsFilters): DBMetric => Rep[Option[Boolean]] = metric => checkMetricsFilters(metric)
+  def originFilters(implicit filters: Filters): DBMetric => Rep[Option[Boolean]] = metric => combineFiltersForOrigin(metric)
```

Comment:
I think the `checkOriginFilters` is better because it is checking specific origin filters